src/main/java/no/nav/pto/veilarbportefolje/huskelapp/controller/dto/HuskelappResponse.java

```diff
@@ -0,0 +1,23 @@
+package no.nav.pto.veilarbportefolje.huskelapp.controller.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;
+import no.nav.common.types.identer.EnhetId;
+import no.nav.common.types.identer.Fnr;
+
+import java.time.LocalDate;
+
+public record HuskelappResponse(
+    String huskelappId,
+    Fnr brukerFnr,
+    EnhetId enhetId,
```

Comment:
Maybe we don't need those fields in response.
```suggestion
```

Reply:
Når vi henter basert på veileder burde vi vel ha fnr i alle fall? for da henter man en liste med alle på en veileders oversikt. så for at vi skal kunne plassere den til riktig brukerrad så må vi ha noe å matche på